### ISSUE TITLE
[Support request] How to bundle extra languages in Webpack without Babel

### ISSUE THREAD

**Issue by peterpeterparker** [PrismJS/prism#1613]
Hi,

Sorry for opening a support request here, I would honestly understand if you close it straight away.

Anyway I try to include more languages than the default one, for example `java`, in a project where I use Webpack but without Babel. Is there somewhere any instructions or example on how to achieve that?

I guess that scripts have to be copied manually or with the help of the `copy-webpack-plugin` to the bundled/dist directory but not sure which scripts (main script, core, which languages, etc.).

If someone could point me to an example that would be cool

Best
David

**Comment by mAAdhaTTah** [comment]
If you're just trying to add more languages, you can import / require them manually, e.g. `require('prismjs/components/prism-java');`. You'll also want to import `prism-core` from the components folder first. If it helps, the [fixtures in the babel plugin](https://github.com/mAAdhaTTah/babel-plugin-prismjs/tree/master/test/fixtures) may be insightful here.

I'm going to close, as this isn't an issue, but feel free to follow-up if you have additional questions.

**Comment by peterpeterparker** [comment]
@mAAdhaTTah thx for pointing me in the right direction. Actually I already did load correctly the extra languages but I faced a problem because I include `prismjs` in a Web Component where I don't want to bundle extra languages but only load them if necessary in the projects where the component is used.

Anyway I think I found a work around, according the user specified language the component will inject and load the scripts from unpkg (https://github.com/fluster/deckdeckgo/blob/master/src/components/slides/deckdeckgo-slide-code/deckdeckgo-slide-code.tsx)

Thx again for the answer and library.

Btw. I quoted you in the documentation of my component (https://github.com/fluster/deckdeckgo/blob/master/doc/slides/slides.md)
